app/config/features/Features.scala

```diff
@@ -28,4 +28,5 @@ class Features @Inject()(config: Configuration) {
   val enableVatReturnsService = new Feature(ConfigKeys.useVatReturnsService, config)
   val enableVatObligationsService = new Feature(ConfigKeys.useVatObligationsService, config)
   val future2020DateEnabled = new Feature(ConfigKeys.future2020DateEnabledFeature, config)
+  val enableLanguageSelector = new Feature(ConfigKeys.useLanguageSelectorFeature, config)
```

Comment:
This feature switch needs adding to the feature switch test only view. You'll need to update the model and form as well.